src/cmd/tools/common.go

```diff
@@ -31,3 +33,15 @@ var toolsCmd = &cobra.Command{
 func Include(rootCmd *cobra.Command) {
 	rootCmd.AddCommand(toolsCmd)
 }
+
+// newVersionCmd is a generic version command for tools
+func newVersionCmd(name, version string) *cobra.Command {
+	return &cobra.Command{
+		Use:   "version",
+		Args:  cobra.NoArgs,
+		Short: lang.CmdToolsVersionShort,
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
```suggestion
		Run: func(cmd *cobra.Command, _ []string) {
```
GH CodeQL lint finding

Reply:
Done